benchmark/aio/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "aio-benchmark",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "initialize": "yarn clone && yarn setup && yarn update", 
+    "clone": "(git clone https://github.com/angular/angular || true) && cd angular && git fetch origin && git reset --hard dd2a650c3455f3bc0a88f8181758a84aacb25fea",
+    "setup": "cd angular && yarn && cd aio && yarn && yarn setup",
+    "//": "Shouldn't need to install the package twice, but the first install seems to leave two @ngtools/webpack installs around.",
+    "update": "cd angular/aio && yarn add ../../../../dist/_angular-devkit_build-angular.tgz --dev && yarn add ../../../../dist/_angular-devkit_build-angular.tgz --dev",
+    "benchmark": "cd angular/aio && benchmark --verbose -- yarn ~~build --configuration=stable"
```

Comment:
what is this `~~` thing?

Reply:
It's a script in AIO:
```
"~~build": "ng build",
```
It's one of the only ones that doesn't have pre and post scripts that would add noise to the benchmark.